ocfweb/docs/docs/staff/policy.md

```diff
@@ -0,0 +1,83 @@
+
+
+Staff members shall be chosen by the OCF Decision Making Process. [per OCF
+Board decision on 4/13/89]
+
+
+## Background
+
+The OCF Decision Making Process spells out the powers of BoD, the GMs, and the
+SMs; however, it doesn’t explain what powers other staffers have. The main
+purpose of this policy is to delineate what things staff can do on their own,
+and what things staff require special permission from the SM to do.
+
+## Authorization
+
+Staff is authorized to act independently of the SMs unless otherwise specified
+here or elsewhere. This is so decisions do not have to be delayed by the
+bureaucracy of having to ask the SMs for approval for all decisions.
+
+## Root Privileges
+
+The Site Managers have the sole power to add staff to the `ocfroot` group,
+otherwise grant root privileges on a machine which runs a production service or
+which mounts NFS, create and grant privileges to `/admin` principals, and grant
+other powers which can be used to gain the aforementioned privileges or access
+other OCF members' data.
+
+Staffers in the `ocfroot` group are known as **Technical Managers**.
+
+To be eligible for Technical Manager status, one must fulfill the following
+critera:
+
+* A Site Manager must be able to recognize you in person
+* You must, generally speaking, have been a staffer for at least a semester
+already
+* You must demonstrate a concrete, actual need for root privileges (e.g. a
+command you need to run which is blocking your OCF work)
+
+The Site Managers can revoke Technical Manager status. This can happen, for
+example, if root privileges aren't being actively used.
+
+If you are a Technical Manager, use your additional powers wisely. Remember
+that you must respect the privacy of other members. In particular:
+
+* Don't read or modify a member's files or directories unless one of the
+following is true:
+  * The file or directory's permissions ordinarily grant you access
+  * The member has requested it (explicitly or implicitly)
+  * You have probably cause to suspect that the member may be involved in some
+policy violation, e.g., a security breach.
+* Don't surveil member's private activity on the OCF, e.g. by strace'ing their
+processes or recording their network traffic, etc.)
+* If you do have to access a member's files or other private data, limit your
+accesses to the minimum required to perform your duties
+* Let other members know when possible if modifying their files
+
+Here are some additional guidelines to follow:
+* If you don't have to use root for something, don't use it
+* Avoid `su`/`sudo`ing as other users
+* Avoid using a root shell (`sudo -i`)
+
+### Abuse of root privileges
+
+The worst thing an OCF staffer could do is abuse root privileges. Abuse of root
+privileges constitutes anything done using root privileges which is in
+violation of OCF policies or applicable laws. Examples of this are violating
+members’ privacy without reasonable cause, improperly modifying files,
```

Comment:
If there's potential for confusion I'd rather change it to "modifying files without authorization" or similar.